be/src/exprs/block_bloom_filter.hpp

```diff
@@ -202,6 +205,13 @@ class BlockBloomFilter {
         return (static_cast<uint64_t>(hash) * m + a) >> 32U;
     }
 
+#ifdef DISALLOW_COPY_AND_ASSIGN
+#undef DISALLOW_COPY_AND_ASSIGN
+#endif
+#define DISALLOW_COPY_AND_ASSIGN(TypeName) \
+    TypeName(const TypeName&) = delete;    \
+    void operator=(const TypeName&) = delete
+
     DISALLOW_COPY_AND_ASSIGN(BlockBloomFilter);
```

Comment:
```suggestion
   BlockBloomFilter(const BlockBloomFilter&) = delete;
   void operator=(const BlockBloomFilter&) = delete;
```

just expand it, it looks like more simpler.